fhd_core/source_modeling/fhd_diffuse_model.pro

```diff
@@ -52,7 +52,8 @@ IF size(diffuse_spectral_index,/type) EQ 10 THEN BEGIN ;check if pointer type
     print,"A spectral index is defined in the saved diffuse model, but this is not yet supported!"
 ENDIF ;case of specifying a single scalar to be applied to the entire diffuse model is treated AFTER building the model in instrumental polarization
 
-model_stokes_arr=healpix_interpolate(model_hpx_arr,obs,nside=nside,hpx_inds=hpx_inds,from_kelvin=diffuse_units_kelvin,coord_sys=coord_use)
+if ~keyword_set(diffuse_units_kelvin) then from_jy_per_sr=1
```

Comment:
Since this is setting the default value for these parameters I would move it to the top of the file, maybe right under `IF Keyword_Set(flatten_spectrum)`...

Also, this leaves `from_jy_per_sr` undefined if `diffuse_units_kelvin` is set. If we later set default values for these parameters that could cause a bug, so I would recommend being explicit. Something like:
```
IF N_Elements(diffuse_units_kelvin) EQ 0 THEN diffuse_units_kelvin = 0
from_jy_per_sr = Keyword_Set(diffuse_units_kelvin) ? 0:1
```

Reply:
I believe I resolved this but it could use a final review